model_hub/examples/huggingface/text-classification/startup-hook.sh

```diff
@@ -0,0 +1,2 @@
+pip install attrdict model-hub tensorflow-text nltk==3.6.5 textattack==0.3.4 datasets transformers
```

Comment:
attrdict, model-hub, datasets, and transformers are installed automatically in the model-hub-transformers docker image so we shouldn't need to install them as part of startup.

Reply:
:+1: